src/connector/src/sink/postgres.rs

```diff
@@ -522,17 +544,29 @@ impl PostgresSinkWriter {
                 0,
                 "flattened parameters are unaligned"
             );
-            let statement = match op {
-                Op::Insert => create_insert_sql(schema, table_name, rows_length),
-                Op::Delete => create_delete_sql(schema, table_name, pk_indices, rows_length),
+            let statement_str = match op {
```

Comment:
nit: looks same as (or similar to) the snippet above. Consider sharing the code so that there is no need to keep them in sync.

Reply:
Addressed it in https://github.com/risingwavelabs/risingwave/pull/21566/files